### PR TITLE
ignore unused-parameter warning in ROS 1 message headers

### PR DESCRIPTION
**Pull request by dirk-thomas**
Before http://ci.ros2.org/view/packaging/job/ros2_packaging_osx/17/
After http://ci.ros2.org/view/packaging/job/ros2_packaging_osx/19/

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -111,6 +117,9 @@ build_executable(dynamic_bridge src/dynamic_bridge.cpp ROS1_DEPENDENCIES
   DEPENDENCIES generate_factories
   TARGET_DEPENDENCIES ${message_packages})
 
+if(NOT WIN32)
+endif()
+
```
Comment (wjwwood) [inline CMakeLists.txt]: Should this be dropped?

**Comment by wjwwood** [comment]
lgtm other than the comment.

File: CMakeLists.txt (hunk 1 above)
Comment (wjwwood) [inline CMakeLists.txt]: I went ahead an dropped it.

Comment (dirk-thomas) [inline CMakeLists.txt]: Thanks.

**Pull request merged**